pkg/server/server.go

```diff
@@ -354,7 +354,7 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *server.Config) http.Ha
 	if c.FlowControl != nil {
 		workEstimatorCfg := flowcontrolrequest.DefaultWorkEstimatorConfig()
 		requestWorkEstimator := flowcontrolrequest.NewWorkEstimator(
-			c.StorageObjectCountTracker.Get, c.FlowControl.GetInterestedWatchCount, workEstimatorCfg)
+			c.StorageObjectCountTracker.Get, c.FlowControl.GetInterestedWatchCount, workEstimatorCfg, c.FlowControl.GetMaxSeats)
```

Comment:
Based on similar update in k8s.io/apiserver/pkg/server - [Link](https://github.com/kubernetes/apiserver/blob/a78d1b9b03fbae72f895dc85f1bf793e77e7e9c5/pkg/server/config.go#L916)